jsapp/scss/components/_kobo.bem.ui.scss

```diff
@@ -683,4 +683,9 @@ span.svg-icon {
       display: block;
     }
   }
+
+  &.dropzone--fullscreen {
+    @extend .assets-table, .assets-table--fullscreen;
+    position: fixed !important; // Why does the extended position not override?
```

Comment:
It's because dropzone adds itself an inline style attribute with `position: relative;` and inline styles have stronger specificity than classes